include/osg/Texture

```diff
@@ -652,11 +675,9 @@ class OSG_EXPORT Texture : public osg::StateAttribute
         /** Get the internal texture format type. */
         inline InternalFormatType getInternalFormatType() const { return _internalFormatType; }
 
-        /* select the size internal format to use based on Image when available or Texture format settings.*/
-        GLenum selectSizedInternalFormat(const osg::Image* image=0) const;
```

Comment:
Curious why this was removed. Maybe the cpp changes will explain it but I am not seeing this removed from the 3.4 diffs.